### ISSUE TITLE
Convert any remaining mixed content (http/https) to prepare for Chrome change

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#6391]
Chrome will soon start blocking any http content loaded from an https page - not just scripts and iframes, but images, etc. 

https://www.bleepingcomputer.com/news/google/google-chrome-to-gradually-start-blocking-mixed-content-by-default/

We did this for iframes and scripts a while back but we should plan to scan for any `http://` in our codebase and fix it! It could make for good first timer issues, too. The standard GitHub search is not bad for finding things like this, so please paste in code links in the comments as you find them. 

Then, we'll have to develop a script to do this for all site database content as well. That'll be harder. 

Any help appreciated!

**Comment by deboragaleano** [comment]
Could I claim this issue? Thanks!

**Comment by deboragaleano** [comment]
Hi @jywarren. I just did a standard GitHub search and it gives me 78 code results, see below:

![Screen Shot 2019-10-05 at 11 36 40](https://user-images.githubusercontent.com/36022831/66253187-9e4b6d80-e764-11e9-9394-ecf7d0b14ca9.png)

Here are some examples of code links that I found (there are more, of course!):

https://github.com/publiclab/plots2/blob/7c9d0637170ec8d39be8f773c6761a225ce7cf87/static/index.html

https://github.com/publiclab/plots2/blob/5c0d96274bd0be15feba69d8f30093082402ace5/public/422.html

https://github.com/publiclab/plots2/blob/5c0d96274bd0be15feba69d8f30093082402ace5/public/404.html 

However, I'm not so sure how to go about converting or developing a script to fix all these links.

**Comment by nstjean** [comment]
@deboragaleano Can you do a replace -> replace all in your text editor program? I know VSCode has that option.

**Comment by deboragaleano** [comment]
Hi @nstjean, yes sure, thanks for your message. I use VSCode so I'll try to a replace :)

**Comment by IvetaHron** [comment]
Hi @deboragaleano, do you need any help with this issue?

**Comment by deboragaleano** [comment]
Hi @IvetaHron, I've been having issues with my computer so this may take a while. If you want to go ahead, please do ;) 
Thanks!

**Comment by IvetaHron** [comment]
Hi @deboragaleano, thank you. I will start to work on it tomorrow. I will keep you posted.

**Comment by SidharthBansal** [comment]
Thanks for your PR.
I will review it in a minute

**Comment by jywarren** [comment]
This is all really good work. Noting that for some uses you can use "//
publiclab.org/..." and omit "http:" or "https:" and it will default to the
one that's being used on the page you're on.

Also, let's continue to do this in relatively small pieces rather than a
huge find/replace of the whole codebase - that way we can diagnose when/if
things go wrong piece by piece. Thanks to everyone! Great work!!!!

**Comment by IvetaHron** [comment]
Hi @SidharthBansal, @jywarren  thank you for reviewing my PRs. I have created new FTO issue - https://github.com/publiclab/plots2/issues/6834 for GCI, but I could not add the GCI label to it. I will continue creating more new issues in coming days.

**Comment by SidharthBansal** [comment]
Sorry for delayed reply. You can surely create new ftos.
Thanks for your support.

**Comment by Karska-dev** [comment]
Hi! Can I work with this?

**Comment by noi5e** [comment]
Hi, I just wanted to give a heads-up and say that I'm going to make some FTO issues out of this one... 

There are still a few places in the codebase with `http` links. Some of them seem to be irrelevant to this issue. _(Like, one of them seems to be a footer in an automated email from the server, and there are a few that are simply Ruby and JS commented code snippets.)_

However there are a few I'm thinking would be good to update to https.

I was thinking I would make a single issue listing out multiple places in the app, so multiple first-timers can link and pull to just a single issue. (In the style of something like issue 8545).

(Outreachy applicant here, and pretty new to Public Lab's codebase, so open to pointers!)

**Comment by jywarren** [comment]
Just noting this is a great idea, and plenty of places to do this! For some cases, you can use `//` instead of `https://` and it will just default to whatever the current page is using. But that won't work in, for example, an email template.

Anyways, this is awesome, thanks!!!

**Comment by ebarry** [comment]
Thanks @noi5e ! Looks like your new issue got a lot of uptake with 5+ contributors writing code towards it, thank you for unblocking these contributions!
Once that issue is closed, let's be sure to come back here and close this one too :)

**Comment by jywarren** [comment]
I just went through https://github.com/publiclab/plots2/search?q=http:// and didn't find any that look like an issue! Closing this! 🎉
